### PR TITLE
Moved combat shotguns to shotgun/automatic/combat

### PR DESCRIPTION
**Pull request by Thunder12345**
shotgun/automatic/ will pump itself automatically on firing
Combat shotgun capacity nerfed to 6 rounds
Replaced all instances of the old path with the new one

### CODE REVIEW

**Comment by phil235** [comment]
Welp, apparently there is a combat shotgun on metastation map.

**Comment by Thunder12345** [comment]
Getting this error every time I try to open meta, so I'm not sure how to actually fix it
http://puu.sh/islkP/6e9a0b021b.jpg

**Comment by Thunder12345** [comment]
Managed to fix it by opening the DMM in notepad, turns out the combat shotgun is referenced in a list for a loot spawner

File: code/modules/projectiles/guns/projectile/shotgun.dm [hunk 1]
```diff
@@ -293,3 +283,17 @@
...
+
+/obj/item/weapon/gun/projectile/shotgun/automatic/shoot_live_shot(mob/living/user as mob|obj)
+	..()
+	src.pump(user)
+
+// COMBAT SHOTGUN //
+
+/obj/item/weapon/gun/projectile/shotgun/automatic/combat
+	name = "combat shotgun"
+	desc = "A semi automatic shotgun with tactical furniture and an six-shell capacity underneath."
```
Comment (phil235) [inline code/modules/projectiles/guns/projectile/shotgun.dm]: "an six-shell" -> "a six-shell"

**Comment by phil235** [comment]
Wait, don't you mean projectile/automatic/shotgun instead?

**Comment by Thunder12345** [comment]
automatic/shotgun is an automatic weapon, and needs a magazine, shotgun/automatic still has the shotgun features, like pumping and the internal magazine. Seemed neater to me to use shotgun/automatic/combat in case we want to add any other, non combat automatic shotguns.

**Comment by Supermichael777** [comment]
if changing sprites its nice to have a comparison of old and new
based on real life all shotguns that aren't the double barrel should do this. it might even make the ma viable choice compared to the lasers. and before the inevitable complaint about the automatic syndicate shotgun, its advantage is reload speed and those silly fire rounds, although giving it burst fire would make a great mob clearer.

**Comment by Lumipharon** [comment]
Metastation has a chance to spawn a combat shotgun in the armory contraband locker.

**Comment by Thunder12345** [comment]
Yeah, I managed to find that by diving into the map in notepad and changing the path manually. DM doesn't like not being able to find a path referenced in a var of an atom that's in the map.

**Pull request merged**